docs/tutorial/quickstart.md

```diff
@@ -12,16 +12,15 @@ Create a new Django project named `tutorial`, then start a new app called `quick
 
     # Create a virtualenv to isolate our package dependencies locally
     virtualenv env
-    source env/bin/activate  # On Windows use `env\Scripts\activate`
+    . env/bin/activate  # On Windows use `env\Scripts\activate`
 
     # Install Django and Django REST framework into the virtualenv
-    pip install django
-    pip install djangorestframework
+    pip install django djangorestframework
 
     # Set up a new project with a single application
-    django-admin.py startproject tutorial .  # Note the trailing '.' character
+    django-admin startproject tutorial .  # Note the trailing '.' character
```

Comment:
Omitting the `.py` is the [same style as they use in the Django tutorial](https://docs.djangoproject.com/en/1.10/intro/tutorial01/#creating-a-project), so yup, I think this is good.